_posts/_typos.toml

```diff
@@ -0,0 +1,3 @@
+[default.extend-words]
```

Comment:
Nice job! I've never added one of these before and it looks like you did it perfectly!

Reply:
Woohoo!